debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbOffsetContext.java

```diff
@@ -107,6 +102,16 @@ public void event(DataCollectionId collectionId, Instant timestamp) {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Get a {@link ReplicaSetPartition} instance for a given {@link ReplicaSet}.
+     *
+     * @param replicaSet the replica set; must not be null.
+     * @return a replica set partition; never null.
+     */
+    public ReplicaSetPartition getReplicaSetPartition(ReplicaSet replicaSet) {
```

Comment:
This method will complement `getReplicaSetOffsetContext` until the connector is refactored to use the multi-partition API ([DBZ-3703](https://issues.redhat.com/browse/DBZ-3703)).